platform/_partials/install/external-access-nginx-loft-values.mdx

```diff
@@ -1,9 +1,9 @@
-To update the vCluster Platform deployment to use the NGINX ingress controller, we can create a simple values
-file that tells vCluster Platform to do just that. If you created a values file during your initial
+To update the platform deployment to use the NGINX ingress controller, create a simple values
```

Comment:
This explanation doesn't really align with how we've been talking about values.yaml and how it is used in the platform chart and should be updated to make it more clear.

Reply:
Capturing feedback here: https://linear.app/loft/issue/DOC-304/custom-domain-configuration-improvements